### PR TITLE
integration/k8s: Allow for pod removal

### PR DESCRIPTION
**Pull request by Jakob-Naucke**
After `kubeadm reset`, we frequently see remaining pods running. Use
waitForProcess with 30 seconds (a bit over the highest I've seen in the
wild) after running all tests to allow for the completion of pod
removal.

Fixes: #3998
Signed-off-by: Jakob Naucke <jakob.naucke@ibm.com>

Respin of #4291

### CODE REVIEW

**Review by jodh-intel — DISMISSED** [review]
Thanks @Jakob-Naucke.

lgtm

**Comment by Jakob-Naucke** [comment]
1. `[ $a > $b ]` is syntactically correct, but doesn't do numeric comparison.
2. I need to check `== 1` anyway. Little tired yesterday.

/test-s390x

**Comment by Jakob-Naucke** [comment]
Except that `No pods found in default namespace.` goes to STDERR. /test-s390x

**Comment by fidencio** [comment]
A head's up that may affect your PR.

In the Architecture Committee meeting from January 25th, 2022, the Architecture Committee has agreed on using the "Dismiss stale pull request approvals when new commits are pushed" configuration from GitHub.  It basically means that if your PR has been rebased or updated, the approvals given will be erased.

In order to minimize traumas due to the new approach, please, consider adding a note on the changes done before the rebase / force-push, and also pinging the reviewers for a subsequent round of reviews.

Thanks for your understanding!

Related issue: https://github.com/kata-containers/community/issues/249

**Comment by Jakob-Naucke** [comment]
condition failed in PVC test /retest-s390x

**Comment by Jakob-Naucke** [comment]
Very unlikely to fix anything. Closing.

**Pull request closed without merging**
